### DISCUSSION TITLE
Kiếm điểm sáng tạo nè

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#12] [Ideas]
Mấy bà ơi, tình hình là báo cáo đã tương đối hoàn chỉnh. 
Nhưng mà tui thấy bài báo cáo hơi đơn điệu, chưa có hình ảnh hay hoa văn trang trí gì hết. Hong biết có app nào chỉnh file word đẹp đẹp để mình có thêm điểm cộng hong taaa

**Comment by Lenga10A2** [comment]
hmmm cái này chắc @thaopham2501 biết á

**Reply by thaopham2501** [reply]
jztr :') tui hong biết, xài word thì tải mấy cái template đẹp đẹp về thử coi???

**Reply by baonhu1304** [reply]
Tải seo tải seo chỉ zới
